src/widgets/Tabs/PatientSelect.js

```diff
@@ -184,17 +151,14 @@ PatientSelectComponent.propTypes = {
   patients: PropTypes.object.isRequired,
   onFilterData: PropTypes.func.isRequired,
   createPatient: PropTypes.func.isRequired,
-  onSortData: PropTypes.func.isRequired,
-  sortKey: PropTypes.string.isRequired,
-  isAscending: PropTypes.bool.isRequired,
   onCancelPrescription: PropTypes.func.isRequired,
 };
 
 export const PatientSelect = connect(mapStateToProps, mapDispatchToProps)(PatientSelectComponent);
 const localStyles = StyleSheet.create({
   container: {
-    height: '75%',
-    marginBottom: 20,
+    flex: 12,
+    height: '100%',
```

Comment:
I have litterally no idea what im doing with styles I just type things until it works

Reply:
looks ok, that's all that matters.